product-mini/platforms/esp-idf/main.c

```diff
@@ -2,38 +2,109 @@
  * Copyright (C) 2019 Intel Corporation.  All rights reserved.
  * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
  */
-
+#include <stdio.h>
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "wasm_export.h"
+#include "aot_export.h"
 #include "bh_platform.h"
+#include "platform_api_vmcore.h"
```

Comment:
Don't include platform_api_vmcore.h, it should have been included when bh_platform.h is included.